.github/workflows/release.yaml

```diff
@@ -41,6 +41,14 @@ jobs:
         profile: minimal
         override: true
 
+    - name: Install Rust toolchain (beta)
+      if: matrix.os == 'macos-latest'
+      uses: actions-rs/toolchain@v1
+      with:
```

Comment:
The `with` config for `actions-rs/toolchain` to cross-compile for Apple Silicon on macOS would need another config item, `target: aarch64-apple-darwin`. This would install the target through `rustup target add` in addition to the native target (`x86_64-apple-darwin`).

Reply:
Oof, I meant to add it, but forgot. Thanks again.